### PR TITLE
Verify SBOM for static binary bundle

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:
We now verify the SBOM for its signature and validity within the get script.

#### Which issue(s) this PR fixes:

None

#### Special notes for your reviewer:
Requires https://github.com/cri-o/cri-o/pull/5988 and https://github.com/kubernetes-sigs/bom/pull/123
#### Does this PR introduce a user-facing change?

```release-note
Updated `get` script to verify the SBOM if the [bom tool](https://github.com/kubernetes-sigs/bom) is available in `$PATH`.
```

### CODE REVIEW

**Comment by saschagrunert** [comment]
Ready for review, PTAL @haircommander

File: scripts/release-notes/release_notes.go [hunk 1]
```diff
@@ -112,9 +112,31 @@ The release notes have been generated for the commit range
...
+  - [cri-o.arm64.%s.tar.gz.spdx](https://storage.googleapis.com/cri-o/artifacts/cri-o.arm64.%s.tar.gz.spdx)
+
+To verify the artifact signatures via [cosign](https://github.com/sigstore/cosign), run:
+
+`+"```"+`console
+> export COSIGN_EXPERIMENTAL=1
+> cosign verify-blob cri-o.amd64.%s.tar.gz \
+    --signature cri-o.amd64.%s.tar.gz.sig \
+    --certificate cri-o.amd64.%s.tar.gz.cert
+`+"```"+`
```
[the remark this answers is not in the document]
Comment (saschagrunert) [reply scripts/release-notes/release_notes.go:132]: Those are separated, but stored side by side in the same bucket. Means that we have possible attack vectors if the bucket itself is compromised.

**Comment by haircommander** [comment]
I suppose it does increase the complexity of attacking, but there's still a route we need to thnk about

/lgtm

**Pull request merged**